cmd/ddev/cmd/get_test.go

```diff
@@ -130,7 +133,7 @@ func TestCmdGetComplex(t *testing.T) {
 	_, err = os.Create(app.GetConfigPath("extra/no-ddev-generated.txt"))
 	require.NoError(t, err)
 
-	out, err := exec.RunHostCommand(DdevBin, "get", filepath.Join(origDir, "testdata", t.Name(), "recipe"))
+	out, err := exec.RunHostCommand(DdevBin, "get", filepath.Join(origDir, "testdata", "TestCmdAddonComplex", "recipe"))
```

Comment:
Use the new add-on command throughout.

Reply:
I've mentioned this a few times now but I'll say it again.
I've intentionally kept these tests exactly as they are. They're intentionally testing `ddev get` so that we know that the proxy is working the way we expect it to. This will be especially important if functionality is added to or changed in the various `ddev add-on` sub-commands.

The tests in this file have all been copied into `addon_test.go` and `addon-get_test.go`, with appropriate names.

If you like, I can delete these old ones - but I think it makes sense to keep them until you eventually remove `ddev get`, or else you don't actually know that `ddev get` is still working as a proxy for `ddev add-on`.